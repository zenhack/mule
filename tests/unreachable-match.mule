#XFAIL: match not implemented
match Ok {} with
  | Ok x -> x
  | Ok y -> {}
end
