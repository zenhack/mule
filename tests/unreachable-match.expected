Unreachable match cases at:

  - line 3, columns 8-9

